scripts/netclient.sh

```diff
@@ -24,6 +24,8 @@ echo "[netclient] starting netclient daemon"
 /root/netclient $VERBOSITY_CMD install
 wait $!
 
+sleep 5
+
```

Comment:
@yabinma, how does adding sleep help fix the issue?

Reply:
There's no obvious error message for the issue. It should be a concurrency issue. Without the sleep, the install is not completed yet when the 'join' starts. In the situation, the endpoint IP is not updated into the config file properly(empty in the observation). So when 1 minute check-in is triggered, there's an endpoint IP update.

With the sleep, the install completes before joining.There is no issue reported during the test.